src/components/redux/reducers/index.jsx

```diff
@@ -1,16 +1,65 @@
 import { combineReducers } from "redux";
+import {
+  LOG_IN,
+  LOG_OUT,
+  LOGIN_ERROR,
+  SHOULD_POP_UP_CLOSE,
+  CURRENT_BOARD_ID,
+} from "../actions/actionsTypes";
 
 const currentBoardIDReducer = (state = [], action) => {
   switch (action.type) {
-    case "CURRENT_BOARD_ID":
+    case CURRENT_BOARD_ID:
       return action.payload;
     default:
       return state;
   }
 };
 
+const handleLogin = (state = {}, action) => {
+  switch (action.type) {
+    case LOG_IN:
+      return {
+        ...state,
+        userName: action.user,
+        isLoggedIn: true,
+        isAdmin: action.admin,
+        authError: false,
+      };
+    case LOG_OUT:
+      return {
+        ...state,
+        userName: {},
+        isLoggedIn: false,
+        isAdmin: false,
+        authError: false,
+      };
+    case LOGIN_ERROR:
+      return {
+        ...state,
+        authError: true,
+        errorDetail: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
+const handlePopUp = (state = {}, action) => {
+  switch (action.type) {
+    case SHOULD_POP_UP_CLOSE:
+      return {
+        isPopUpClosed: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
 const allReducers = combineReducers({
   currentBoardIDReducer,
+  handleLogin,
```

Comment:
i see what went on here. usually when we combinereducers, they should not be called with handleX etc. handleX means a function.

when you combine reducers, they should be indicative of the "domain" they are in. to give you an example, my app has:
```
rootState.auth
rootState.card
rootState.study
```

the names are indicative of a "category", the purpose of combining reducers is to allow us to organize and divide. so please rename these